ui/src/components/post-form.tsx

```diff
@@ -151,7 +151,7 @@ export class PostForm extends Component<PostFormProps, PostFormState> {
 
   componentWillUnmount() {
     this.subscription.unsubscribe();
-    this.choices && this.choices.destroy();
+    /* this.choices && this.choices.destroy(); */
```

Comment:
Committed by accident?

Reply:
Its in a separate commit, I think I accidentally added it to this branch, its okay, its a minor bugfix.